cmd/argo/commands/lint.go

```diff
@@ -37,32 +38,41 @@ func NewLintCommand() *cobra.Command {
 						return err
 					}
 				}
-				fmt.Printf("%s is valid\n", file)
+				fmt.Printf("%s is valid \u2713\n ", file)
 				return nil
 			}
-
 			for _, file := range args {
 				stat, err := os.Stat(file)
 				errors.CheckError(err)
 				if stat.IsDir() {
-					err := filepath.Walk(file, func(path string, info os.FileInfo, err error) error {
+					filepath.Walk(file, func(path string, info os.FileInfo, err error) error {
 						fileExt := filepath.Ext(info.Name())
 						switch fileExt {
 						case ".yaml", ".yml", ".json":
 						default:
 							return nil
 						}
-						return lint(path)
+						err = lint(path)
+						if err != nil {
+							unvalidWf = fmt.Errorf("unvalid workflow/workflows found")
+							log.Warn(fmt.Errorf("%w \u2717", err))
```

Comment:
We don't want to use Unicode characters, reporting the raw error is fine.

What would be nice is if you prepended the error with the file name. Something like `workflow1.yaml: this is the error message`

Reply:
The current behaviour is: 
``` 
WARN[0000] /Users/niklashansson/Documents/go/src/github.com/argoproj/argo/test/e2e/functional/steps.yaml failed to parse: error unmarshaling JSON: while decoding JSON: json: unknown field "kiiind" 
```

Reply:
Ah got it. It's fine as is then 🙂